system-test/storage.ts

```diff
@@ -3022,11 +3018,10 @@ describe('storage', () => {
     });
 
     describe('second service account', () => {
-      before(async function () {
+      before(function () {
```

Comment:
We do not seem to clean up Hmac keys from the `SECOND_SERVICE_ACCOUNT`?

Should add the logic?
1. Capture `accessId` in the `'should create key for a second service account'` test
2. Delete the Hmac key by accessId in the `after`

WDYT?

Reply:
Done!!